### ISSUE TITLE
Color selection in type annotation

### ISSUE THREAD

**Issue by bmadone** [drcmda/poimandres-theme#19]
When I select text in suggestion it has the same color as selection color
<img width="778" alt="CleanShot 2022-11-05 at 15 13 17@2x" src="https://user-images.githubusercontent.com/23120972/200121695-deef943e-4d1c-4db6-b84c-c3bdfdd31e16.png">

<img width="730" alt="CleanShot 2022-11-05 at 15 14 55@2x" src="https://user-images.githubusercontent.com/23120972/200121726-ea3fbb7e-7fd3-4e02-ab42-fc634f9106a2.png">

**Comment by imhalid** [comment]
yes there are such annoying colour distortions at some points, for these I edited my `setting.js` file as follows: 

```js
  "workbench.colorCustomizations": {
    "[poimandres-noitalics]": {
      "peekViewEditorStickyScroll.background": "#1C1E28",
      "peekView.border": "#59d9bd70",
      "peekViewEditor.background": "#232531",
      "selection.background": "#59d9bd46"
    }
  }
```
